app/views/layouts/mailer.html.haml

```diff
@@ -61,7 +61,7 @@
             %table.main{:bgcolor => "#fff", :cellpadding => "0", :cellspacing => "0", :style => "font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; border-radius: 3px; background-color: #fff; margin: 0; border: 1px solid #e9e9e9;", :width => "100%"}
               %tr{:style => "font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;"}
                 %td.alert.alert-warning{:align => "center", :bgcolor => "#CE2029", :style => "font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 16px; vertical-align: top; color: #fff; font-weight: 500; text-align: center; border-radius: 3px 3px 0 0; background-color: #CE2029; margin: 0; padding: 20px;", :valign => "top"}
-                  = I18n.t("association_name")
+                  = I18n.t('association_name')
```

Comment:
Maybe create a rule / test that checks this kind of formatting issues in CircleCI

Reply:
It's possible, but it would take a lot of time to fix throughout the application. I've only fixed a few in this PR, since they were giving me errors. So as far as I know, I think there might be a test for this for new translations, but not old ones